lib/engine/round/operating.rb

```diff
@@ -304,6 +305,8 @@ def _process_action(action)
         when Action::Bankrupt
           liquidate(entity.owner)
         end
+
```

Comment:
is it possible for the operating info to get overidden to the wrong thing? this looks a bit unsafe

Reply:
That is exactly the idea that each method would overwrite its own operating info. The default is to withhold with 0 revenue.

Reply:
I think this is the simplest solution but if you have a better idea on how to do this, I am all ears :)

Reply:
if you rebase master, you'll see a new method called start_operating

in there you can initialize yoru dummy 0 value

then only in dividend, you can add that, much safer

Reply:
perfect will do!